docs/guide/cell-references.md

```diff
@@ -160,7 +160,38 @@ copied between different places.
   </tbody>
 </table>
 
-## Circular reference
+## Range references
+
+In HyperFormula, a range is a group of two or more adjacent cells.
+
+You can reference ranges in the following ways:
+
+| Range reference type | Example | Reversed examples       |
+| -------------------- | ------- | ----------------------- |
+| Cell addresses       | A1:B2   | A2:B1<br>B1:A2<br>B2:A1 |
+| Columns              | A:B     | B:A                     |
+| Rows                 | 1:2     | 2:1                     |
```

Comment:
I'd rather describe these as separate types of range than ways of refencing.

- Cell ranges are always in shape of a finite rectangle
- Column ranges contain the whole columns (they are infinite in the row dimension)
- Row ranges contain the whole rows